files/fr/web/css/_doublecolon_target-text/index.html

```diff
@@ -0,0 +1,43 @@
+---
+title: '::target-text'
+slug: 'Web/CSS/::target-text'
+browser-compat: css.selectors.target-text
+translation_of: 'Web/CSS/::target-text'
+---
+<div>{{CSSRef}}{{SeeCompatTable}}</div>
+
+<p>Le <a href="/fr/docs/Web/CSS/Pseudo-elements">pseudo-élément</a> <a href="/en-US/docs/Web/CSS">CSS</a> <strong><code>::target-text</code></strong> représente le texte vers lequel l'écran vient de défiler, dans le cas ou le  navigateur supporte les fragments <code>scroll-to-text</code>. Il permet aux auteurs de mettre en valeur cette section de texte.</p>
```

Comment:
"ou" > "où"
"supporte" > "prend en charge" (pour ce dernier, dans la mesure du possible où ça n'alourdit pas trop la phrase).

Reply:
> "supporte" > "prend en charge" 

Ah cool, we have the same rule at WordPress. But as I seen "Supporte" in another MDN docs, I just followed the example. I can't tell where I found it though 🤷 

Fixed in https://github.com/mdn/translated-content/pull/1720/commits/6483766c8e25439354b06ef6e3c7be57bee0efb3